stethoscope/ui/src/CriticalDeviceSummary.js

```diff
@@ -1,19 +1,24 @@
 import React from 'react'
 import { criticalDeviceSummary } from './utils/device-filter'
+import Accessible from './Accessible'
 
 export default (props) => {
   const devices = props.devices || props.store.devices
   if (!devices) return null
   const summary = criticalDeviceSummary(devices)
 
   if (summary) {
-    return <p>
-      You have <span className='text-danger'>{summary.count}</span> {summary.pluralizedDevices} that {summary.pluralizedRequires} attention:
-      &nbsp;
-      <span className='text-danger'>
-        {summary.devicesList}
-      </span>
-    </p>
+    return (
+      <Accessible role='marquee' live>
```

Comment:
Not sure I understand this one. Shouldn't `marquee` only be for scrolling text? Also, looking at [this page](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions) makes me think the `live` attribute should have a value of `off`, `polite`, or `assertive`.